include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -355,7 +359,8 @@ namespace VectorTools
         Assert (touch_count[i] != 0,
                 ExcInternalError());
 
-        data_2(i) /= touch_count[i];
+        ::dealii::internal::ElementAccess<OutVector>::multiply(
+          1./touch_count[i], i, data_2);
```

Comment:
Since `multiply` is only used twice (here and in `fe_tools`), I would prefer not to do it via a separate function but rather `get` and `set` combined.